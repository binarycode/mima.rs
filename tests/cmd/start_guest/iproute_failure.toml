args = 'start-guest zero'
status.code = 1
