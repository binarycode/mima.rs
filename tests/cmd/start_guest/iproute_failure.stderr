error: Failed to run 'ssh -o BatchMode=yes -o ConnectTimeout=10 -o ForwardAgent=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@10.210.0.13 ip link set mima-mgt-zero master mima-mgt up'

stdout:
foobar

